react/package.json

```diff
@@ -24,22 +24,23 @@
     "vtex.checkout-summary": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-summary@0.22.0/public/@types/vtex.checkout-summary",
     "vtex.css-handles": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.css-handles@1.0.0/public/@types/vtex.css-handles",
     "vtex.device-detector": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.device-detector@0.2.6/public/@types/vtex.device-detector",
-    "vtex.flex-layout": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.flex-layout@0.20.1/public/@types/vtex.flex-layout",
+    "vtex.flex-layout": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.flex-layout@0.21.0/public/@types/vtex.flex-layout",
+    "vtex.minicart": "https://natandevuamerge--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.minicart@2.66.0+build1680722065/public/@types/vtex.minicart",
```

Comment:
I've just tried to `yarn install` inside`react/` and got this error:

```
yarn install v1.22.19
warning package.json: No license field
warning vtex.minicart@2.66.0: No license field
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[#####################################################] 694/695de .
error https://natandevuamerge--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.minicart@2.66.0+build1680722065/public/@types/vtex.minicart: Integrity check failed for "vtex.minicart" (computed integrity doesn't match our records, got "sha512-W7JT0bJj2oZrL0EQwEJ49OlRGHG209GbVI8+0CsaUy7WOPBP+lmS5vP4u1+gRN/ZQe/XsURfbsaFuBk76AE0EA== sha1-s7PrrJlcn69W3quEiKSQmCi46HQ=")
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

I see it's trying to install the same package from that `natandevuamerge--storecomponents` workspace.

I believe this line got added automatically when you ran `vtex setup`, but I think we can safely remove it.